src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/DirectoryDAO.scala

```diff
@@ -40,4 +40,5 @@ trait DirectoryDAO {
   def getPetServiceAccountForUser(userId: WorkbenchUserId): Future[Option[WorkbenchUserServiceAccountEmail]]
   def addPetServiceAccountToUser(userId: WorkbenchUserId, petServiceAccountEmail: WorkbenchUserServiceAccountEmail): Future[WorkbenchUserServiceAccountEmail]
   def removePetServiceAccountFromUser(userId: WorkbenchUserId): Future[Unit]
+  def getUserFromPetServiceAccount(petSA:WorkbenchUserServiceAccountEmail):Future[Option[WorkbenchUser]]
```

Comment:
This is a style thing, but I think we generally prefer to put a space after the colon. So `petSA: WorkbenchUserServiceAccountEmail`